### PR TITLE
DEV: Adding Attenuators

### PR DESCRIPTION
**Pull request by ghalym**
Just like the stoppers and the apertures, all the gas and solid attenuators are added as L_Attenuators and K_Attenuators.

All Unit tests passed.
image

### CODE REVIEW

**Review by klauer — CHANGES_REQUESTED** [review]
Additional enums look fine, minus the one capitalization difference.

What is the .TcTLEO file extension here? I don't recall seeing that - and I think our pre-commit doesn't format those files by default at least.

File: lcls-twincat-pmps/PMPS/DUTs/InternalDUT/K_Attenuators.TcTLEO [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <EnumerationTextList Name="K_Attenuators" Id="{a24423f0-e3ff-4130-a3a6-1a87eb45eaee}">
+    <Declaration><![CDATA[{attribute 'qualified_only'}
+{attribute 'strict'}
+TYPE K_Attenuators :
+(
+	AT1K0 := 1, // FEE Gas Attenuator
+    AT1K4 := 2, // TMO Solid Attenuator in the FEE
+	AT1K2 := 3, // RIX Solid Attenuator
+	AT2k2 := 4, // RIX Solid Attenuator
```
[the remark this answers is not in the document]
Comment (ghalym) [reply lcls-twincat-pmps/PMPS/DUTs/InternalDUT/K_Attenuators.TcTLEO]: I fixed it. And these are internal DUTs

**Review by klauer — APPROVED** [review]

**Comment by ghalym** [comment]
@slacAWallace I made the new files with textlist support this way they are the same as the L_Stopper and K_stopper. Is there a specific reason for having them with textlist support?

**Comment by slacAWallace** [comment]
Textlist support pertains to TC HMI display iirc. Not sure why it's important or enabled for any struct in this project. It may be an artifact of the HMI testing I was doing before using TcUnit.

**Comment by ghalym** [comment]
I just removed all textlist support files. now they are all .TcDUT.

**Comment by klauer** [comment]
Looks better to me, Maggie! Sorry for the extra work :)

File: lcls-twincat-pmps/PMPS/DUTs/InternalDUT/K_Stopper.TcDUT [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+{attribute 'strict'}
+TYPE K_Stopper :
+(
+	ST3K4 := 1, // TMO PPS stopper in
+    ST1K2 := 2, // RIX PPS stopper in
+    MR1K1_IN := 3, // Monochromator focus optic vertical state
+    MR1K1_OUT := 4, // Monochromator focus optic vertical state
+    MR1K3_IN := 5, // TXI Offset Mirror horizontal state
+    MR1K3_OUT := 6, // TXI Offset Mirror horizontal state
+    DEFAULT := PMPS_GVL.MAX_VETO_DEVICES // DO NOT USE FOR ANYTHING
```
Comment (ZLLentz) [inline lcls-twincat-pmps/PMPS/DUTs/InternalDUT/K_Stopper.TcDUT:14]: Why is this here if we aren't supposed to use it?

Comment (ZLLentz) [reply lcls-twincat-pmps/PMPS/DUTs/InternalDUT/K_Stopper.TcDUT:14]: Not important for this PR but just curious

**Review by ZLLentz — APPROVED** [review]

**Pull request merged**